index.js

```diff
@@ -242,9 +249,9 @@ TCurl.prototype.request = function tcurlRequest(opts, delegate) {
         }
     });
 
-    client.waitForIdentified({
-        host: opts.hostname + ':' + opts.port
-    }, onIdentified);
+    var peer = subChan.peers.choosePeer();
```

Comment:
I don't think this really matters for tcurl. Any peer would be fine for the first connection attempt.